internal/xds/translator/cluster.go

```diff
@@ -85,14 +85,32 @@ func buildEndpointType(settings []*ir.DestinationSetting) EndpointType {
 
 func buildXdsCluster(args *xdsClusterArgs) *clusterv3.Cluster {
 	dnsLookupFamily := clusterv3.Cluster_V4_PREFERRED
+	customDNSPTrafficPolicy := args.dns != nil && args.dns.LookupFamily != nil
+	// apply DNS lookup family if custom DNS traffic policy is set
+	if customDNSPTrafficPolicy {
```

Comment:
`customDNSTrafficPolicy` ? is the `T` intentional?

Reply:
Updated